src/main/java/com/atlassian/db/replica/spi/SuppliedCache.java

```diff
@@ -14,9 +14,7 @@
     Optional<T> get(Supplier<T> supplier);
 
     /**
-     * Returns the cached value
-     *
-     * @return
+     * @return the cached value
```

Comment:
isn't it obvious? :)

Reply:
yes, I just moved the obvious comment to fit into the javadoc format